pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -172,6 +173,18 @@ type CSINodeInfo interface {
 	GetCSINodeInfo(nodeName string) (*storagev1beta1.CSINode, error)
 }
 
+var _ CSINodeInfo = &CachedCSINodeInfo{}
```

Comment:
nit: this interface should be renamed to `CSINodeGetter`

Reply:
replied in the other comment.